applications/crossbar/src/crossbar_auth.erl

```diff
@@ -237,25 +202,21 @@ auth_config(AccountId) ->
 -spec account_auth_config(api_ne_binary(), api_ne_binary()) -> kz_json:object().
 account_auth_config('undefined', _MasterId) ->
     system_auth_config();
-account_auth_config(MasterId, ?NE_BINARY = MasterId) ->
+account_auth_config(MasterId, ?NE_BINARY=MasterId) ->
```

Comment:
Please use `?MATCH_ACCOUNT_RAW(MasterId)`. Should help understand things better in the long run, or for newcomers.